rdf/rdf12/rdf-semantics/index.html

```diff
@@ -541,7 +541,17 @@ <h2>
             <dt>type</dt>
             <dd>mf:PositiveEntailmentTest</dd>
             <dt>approval</dt>
-            <dd property='mf:approval' resource=''></dd>
+            <dd property='mf:approval' resource=''>none</dd>
```

Comment:
This may become unnecessary if another PR is merged. This looks like it is the report generator catching up.